source/simulator/core.cc

```diff
@@ -1693,6 +1704,20 @@ namespace aspect
               solve_advection(AdvectionField::composition(c));
             }
 
+          if (
+            parameters.use_discontinuous_composition_discretization
+            &&
+            parameters.use_limiter_for_discontinuous_composition_solution
+          )
+            {
+              pcout << "Apply bound preserving limiter to the composition advection field "
```

Comment:
One more important thing: Please change this line (and the temperature equivalent) to literally: "   Applying bound preserving limiter to the composition/temperature advection fields." That means three spaces up front, changing the "Apply" to "Applying" and changing "field " into "fields." This might seem like nitpicking, but currently this output looks completely out of place compared to the other lines of the screen output, and that makes it harder to read and understand what the program does.